db/schema.rb

```diff
@@ -10,15 +10,35 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_03_15_170837) do
+ActiveRecord::Schema.define(version: 2019_04_10_203341) do
+
+  create_table "active_storage_attachments", force: :cascade do |t|
+    t.string "name", null: false
+    t.string "record_type", null: false
+    t.integer "record_id", null: false
+    t.integer "blob_id", null: false
+    t.datetime "created_at", null: false
+    t.index ["blob_id"], name: "index_active_storage_attachments_on_blob_id"
+    t.index ["record_type", "record_id", "name", "blob_id"], name: "index_active_storage_attachments_uniqueness", unique: true
+  end
+
+  create_table "active_storage_blobs", force: :cascade do |t|
+    t.string "key", null: false
+    t.string "filename", null: false
+    t.string "content_type"
+    t.text "metadata"
+    t.bigint "byte_size", null: false
+    t.string "checksum", null: false
+    t.datetime "created_at", null: false
+    t.index ["key"], name: "index_active_storage_blobs_on_key", unique: true
+  end
 
   create_table "certificates", force: :cascade do |t|
     t.string "value"
     t.string "usage"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.integer "component_id"
-    t.index ["component_id"], name: "index_certificates_on_component_id"
```

Comment:
Why is this being removed?

Reply:
Excellent spot; must have happened when I was squashing commits by mistake.